samples/benchmarks/coremark/sample.yaml

```diff
@@ -14,12 +14,14 @@ tests:
       - nrf5340dk/nrf5340/cpuapp
       - nrf54l15pdk/nrf54l15/cpuapp
       - nrf54l15pdk@0.3.0/nrf54l15/cpuapp
+      - nrf54h20dk/nrf54h20/cpuapp
```

Comment:
I wonder if we should test nrf54h20 for all configurations in every twister run? I suppose some configurations can be omitted on nrf54h20 and only be run on other platform - to do it please add entry in scripts/quarantine_integration.yaml

Also running  sample.benchmark.coremark_static or sample.benchmark.coremark_multithread: 
for nrf53 or nrf54h wouldn't add memory method to remote core. We should make a follow up ticket to change configuration to base it on FILE_SUFIX sysbuild feature. We can then align it with sample.benchmark.coremark_heap

Reply:
https://nordicsemi.atlassian.net/browse/NCSDK-27522. Please do not resolve this conversation to keep the web link active. I put it in the Jira ticket.

Reply:
Then maybe remove running it on older platforms.

Reply:
In order not to have to much configurations build on CI on commit. They will be build anyway nightly.

Reply:
I'd say few more doesn't make a big difference. If a sample grows in configuration number, it should be normal that test scope also grows. Adding new configurations doesn't make older configurations less important.

Reply:
Discussed offline, I added some configurations to quarantine.